teacher.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'person'
+class Teacher < Person
+  attr_reader :salary, :target_raise
+
+  def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
+
+  def salary=(new_salary)
+    "This better be good!"
+    @salary = new_salary
+  end
+
+  def set_performance_rating(rating)
+    response = ""
```

Comment:
This works, but it looks a little odd. Probably OK to just `return` the proper string at the time a particular condition is hit.